checkstyle.properties

```diff
@@ -1,12 +1,18 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18,4.19,4.20
-publicVersions=4.21
+archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18,4.19,4.20,4.21
+publicVersions=4.22
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+4.22.description=Upgrade to Checkstyle 8.22
+4.22.sqVersions=[6.7,LATEST]
+4.22.date=2019-09-13
+4.22.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.22
+4.22.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.22/checkstyle-sonar-plugin-4.22.jar
+
 4.21.description=Upgrade to Checkstyle 8.21
 4.21.sqVersions=[6.7,LATEST]
```

Comment:
You can't have two versions that are compatible with `LATEST`. The convention is to change the end range of the archived version to the version of SQ that's current at the time of the new release.

Reply:
Thank you very much. I‘ve changed it and sorry I missed this one.